test/e2e/benchmark/benchmark.go

```diff
@@ -20,14 +25,30 @@ type BenchmarkTest struct {
 }
 
 func NewBenchmarkTest(name string, manifest *Manifest) (*BenchmarkTest, error) {
-	// create a new testnet
-	testNet, err := testnet.New(context.Background(), name, seed,
-		testnet.GetGrafanaInfoFromEnvVar(), manifest.ChainID,
-		manifest.GetGenesisModifiers()...)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	identifier := fmt.Sprintf("%s_%s", name, time.Now().Format(timeFormat))
+	kn, err := knuu.New(ctx, knuu.Options{
+		Scope:        identifier,
+		ProxyEnabled: true,
+	})
```

Comment:
[nit] could be named `scope` because that's how it is used later

```suggestion
	scope := fmt.Sprintf("%s_%s", name, time.Now().Format(timeFormat))
	kn, err := knuu.New(ctx, knuu.Options{
		Scope:        scope,
		ProxyEnabled: true,
	})
```